src/sagas/wallet.js

```diff
@@ -260,6 +261,17 @@ export function* startWallet(action) {
     });
 
     yield put(setServerInfo(serverInfo));
+
+    let network = get(serverInfo, 'network');
+    if (useWalletService) {
+      // In the wallet-service facade, serverInfo is null, so we need to get
+      // version data
+      const versionData = yield call([wallet, wallet.getVersionData]);
+      network = versionData.network;
+    }
+
+    // Set the network name in redux
+    yield put(setNetworkName(network));
```

Comment:
The `serverInfo` is the same return as in the `versionData`. Why don't you save the `versionData` in the `serverInfo` redux state that we already have instead of creating a new one to save this same information?